src/System.Data.SqlClient/ref/System.Data.SqlClient.NetCoreApp.cs

```diff
@@ -5,6 +5,19 @@
 // Changes to this file must follow the http://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
 
+namespace Microsoft.SqlServer.Server
+{
+    [System.AttributeUsageAttribute((System.AttributeTargets)(10624), AllowMultiple=false, Inherited=false)]
```

Comment:
Can we expand 10624 to read `AttributeTargets.Field | AttributeTargets.Property |
                      AttributeTargets.ReturnValue | AttributeTargets.Parameter`

Reply:
Will do, it would get blown away by the next auto update - except that auto update wouldn't work for this assembly anyway, as the ref is split across files.